Common/src/geometry/meshreader/CSU2ASCIIMeshReaderFVM.cpp

```diff
@@ -34,9 +34,9 @@ CSU2ASCIIMeshReaderFVM::CSU2ASCIIMeshReaderFVM(CConfig* val_config, unsigned sho
       myZone(val_iZone),
       nZones(val_nZone),
       meshFilename(config->GetMesh_FileName()) {
-  actuator_disk = (((config->GetnMarker_ActDiskInlet() != 0) || (config->GetnMarker_ActDiskOutlet() != 0)) &&
-                   ((config->GetKind_SU2() == SU2_COMPONENT::SU2_CFD) ||
-                    ((config->GetKind_SU2() == SU2_COMPONENT::SU2_DEF) && (config->GetActDisk_SU2_DEF()))));
+  actuator_disk =
+      (((config->GetnMarker_ActDiskInlet() != 0) || (config->GetnMarker_ActDiskOutlet() != 0)) &&
+       ((config->GetKind_SU2() == SU2_COMPONENT::SU2_CFD) || ((config->GetKind_SU2() == SU2_COMPONENT::SU2_DEF))));
```

Comment:
It was used here, removing it changes the default of behavior of SU2_DEF when the config contains actuator disks

Reply:
OK, thanks, I will change it back.